src/Api/StructureShape.php

```diff
@@ -24,12 +29,11 @@ public function __construct(array $definition, ShapeMap $shapeMap)
      */
     public function getMembers()
     {
-        $result = [];
-        foreach ($this->definition['members'] as $name => $definition) {
-            $result[$name] = $this->shapeFor($definition);
+        if (empty($this->members)) {
```

Comment:
Oh, right. I find `empty` to be clearer than an exclamation point when reading code, and using empty avoids the implicit cast to boolean that simply calling `if (!$this->members) {` would entail.